sdk/communication/azure-communication-sms/pom.xml

```diff
@@ -55,7 +55,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-communication-common</artifactId>
-      <version>1.1.0-beta.1</version> <!-- {x-version-update;com.azure:azure-communication-common;current} -->
```

Comment:
Not needed as well

Reply:
But 1.1.0-beta.1 doesn't exist, won't the build fail?